src/components/Issues/Issues.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './Issues.css'
+import { Card, Feed, List, Label } from 'semantic-ui-react';
+
+const Issues = () => (
```

Comment:
I know this is just a preliminary pr. But you might wanna create smaller components for a whole bunch of code in this component.